src/components/Modal/Modal.scss

```diff
@@ -5,7 +5,7 @@ $small-width: rem(620px);
 .BodyWrapper {
   display: flex;
   flex-grow: 1;
-  overflow-x: hidden;
+  overflow: hidden;
```

Comment:
why does this need to be changed?

Reply:
Whoops, leftover change from my prev idea